Makefile

```diff
@@ -0,0 +1,40 @@
+###############################################################################
+###                                  Build                                  ###
+###############################################################################
+
+# Check Go version
+check_version:
```

Comment:
This is not needed, since the go version is specified in the `go.mod`, go will download the correct toolchain version to build.
This might also result in problems with go versions higher than 1.20.2 being installed.